transcoder/main.py

```diff
@@ -30,11 +30,11 @@
 import logging
 import os
 
-from transcoder import __version__, LineEncoding
-from transcoder.message.MessageParser import MessageParser
+from transcoder.version import __version__
```

Comment:
This import probably needs to change back to what's on 33 for __version__ for PyPi packaging